templates/performance_test_1p_1k.txt

```diff
@@ -1,5 +1,5 @@
-  - title: Average Round-Trip Time
-    description: "The figure shown above shows the average round-trip time in milisecond for different DDS vendors using a 1K array message. All DDS vendors are configured in asynchronous mode except cycloneDDS which is configured in syncronous mode.<br></br><ul><li><b>QoS</b> Best effort</li><li><b>Rate</b> 1000</li><li><b>Runtime</b> 30 seconds</li><li><b>N process</b> 1</li><li><b>Message size</b> 1K</li></ul>"
+  - title: Latency
```

Comment:
nit: Latency can refer to different things.
I would be more explicit in either the title or the description.

e.g.: Modify the title to `Average single-trip latency`
It's also good to remark that it's an average (IIUC, it's an average. If it's a different statistical value, it should be clear).

Reply:
Applies to other files too.

Reply:
Sorry guys, I'm not having much time these days, so help would be appreciated. I thought about keeping the average term (yes, you understand correctly, it is an average). In the end I took it out just because other data are also averages and the only one saying it was latency, but it could (and probably should) say average or whatever statistic is used for every measurement.